### PR TITLE
Units in CSS

### LINKED ISSUE #27: Look into sizing unit best practices
When should we be using px and % vs rem?

Is it ok to use px for margin and padding etc?

### LINKED ISSUE #51: Use px or em for media queries

### PR DESCRIPTION
**Pull request by austinstarin**
Close #51, Close #27

### CODE REVIEW

File: CSS/overview.md [hunk 1]
```diff
@@ -78,3 +79,16 @@ It's also a helpful pattern to prefix selectors (i.e. `.o-, .c-, .u-`) using the
...
+## Sizing
+
+<a name="sizing"></a><a name="3.1"></a>
+[3.1](#sizing) Sizing
+
+- Use `px` for borders
+- Use unitless line height
+- Use `em` for media queries
+- Use `rem` for everything else (font size, spacing, etc.)
+- `%` is also appropriate and and can be used for column widths, etc.
```
Comment (bgantick) [inline CSS/overview.md]: typo `and and`

**Comment by abea** [comment]
This looks good. The only other use of `em` I've found helpful is on text top or bottom padding--so the padding adjusts to the font size for headings and whatnot.

Are we enforcing (or can we enforce) this in the stylelint config already?

It's not a CSS unit, but considering the subject of relative units maybe we should specify a practice of setting the root value. Either leaving to the browser default or explicitly doing so on `html`.

**Comment by austinstarin** [comment]
Interesting. This is enforced loosely in our stylelint config but could use more close attention.

In reference to font sizes we generally set the root value to `62.5%` and then size in `rems` off of that base value. This is an authoring convenience so that `1.6rem` is equal to `16px` for example.

The decision to set this root value using `%` was so that a user's browser font setting would scale text appropriately.

**Comment by abea** [comment]
Is that (the 62.5%) something that should be here in CSS best practices? Seems worth formalizing if a standard practice.

**Comment by austinstarin** [comment]
Fixed typo @bgantick 51f6f8e

**Pull request merged**